src/server/pfs/server/driver.go

```diff
@@ -113,26 +113,23 @@ const (
 	tombstone = "delete"
 )
 
-// Instead of making the user specify the respective size for each cache,
-// we decide internally how to split cache space among different caches.
-//
-// Each value specifies a percentage of the total cache space to be used.
 const (
```

Comment:
Probably could use a comment here explaining that this is the number of objects we cache.